VideoLocker/res/drawable/list_item_selector.xml

```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:drawable="@color/edx_grayscale_neutral_light" android:state_pressed="true"/>
-    <item android:drawable="@color/white"/>
```

Comment:
Oh, I was actually mis-remembering this bug. What you did is fine, I think.

For the record, the bug I was referencing is when you define a <shape> but don't give it any contents, which causes some Android versions do render as black. (For example, the first issue in this bug report: https://openedx.atlassian.net/browse/MA-1829)